rebar.config.script

```diff
@@ -151,7 +151,7 @@ DepDescs = [
 %% Third party deps
 {folsom,           "folsom",           {tag, "CouchDB-0.8.3"}},
 {hyper,            "hyper",            {tag, "CouchDB-2.2.0-6"}},
-{ibrowse,          "ibrowse",          {tag, "CouchDB-4.0.1-2"}},
+{ibrowse,          "ibrowse",          {tag, "CouchDB-4.4.2-1"}},
```

Comment:
There are dragons there:
- [dump message queue to log](https://github.com/apache/couchdb-ibrowse/blob/CouchDB-4.4.2-1/src/ibrowse.erl#L504)

Reply:
Good idea to check. It looks it won't log unless we configure a logger in the app env:

https://github.com/apache/couchdb-ibrowse/blob/CouchDB-4.4.2-1/src/ibrowse_lib.erl#L443-L446

Reply:
@iilyak would you want to make a PR for upstream to remove that, though with the logging not configured not sure how dangerous it is still?

And we could perhaps also try to upstream the commit to scrub credentials as well. I imagine other users would want to keep credentials not showing up in the logs.

Reply:
I did more thorough review of ibrowse source code. There are couple of potential issues. 1) debug/trace mode and 2) dumping of message queue using log_msg. I do agree with you that neither of these would become a problem if we wouldn't configure logger.